### ISSUE TITLE
Extension triggers highlight in markdown checkboxes

### ISSUE THREAD

**Issue by theodiem** [Gruntfuggly/todo-tree#856]
With the extension:
![image](https://github.com/Gruntfuggly/todo-tree/assets/1006506/6c2f63b7-ed82-497c-9014-885b6e719f3f)

Without it (expected):
![image](https://github.com/Gruntfuggly/todo-tree/assets/1006506/a2cd23ca-ee7b-4da0-a09f-3929f5a0f638)

I believe not every checkbox in a markdown is a TODO list. Maybe I just want a checkbox there to be rendered by some parser.
I hope its just a settings issue, but couldn't find the readme file anything too obvious on how to catch only this in markdown files.

**Comment by theodiem** [comment]
Found references of it on #256 and #577 . IMHO it is a preference thing if a user wants all checkboxes in a markdown to be considered as TODO itens or not. 
I've lookied into default `todo-tree.regex.regex` and what change fixed #577 but the commit isn't linked in the issue.
I've also checked https://github.com/Gruntfuggly/todo-tree?tab=readme-ov-file#markdown-support , but it seems that `[ ]` and `[x]` are not part anymore of the default list of tags - at least not according to the docs - which I could just exclude.

**Comment by theodiem** [comment]
Sorry for the noise. Found a way to solve my issue:
Add to your settings.json:
```
    "todo-tree.regex.regex": "(//|#|<!--|;|/\\*|^|^\\s*(-|\\d+.))\\s*($TAGS)",
    "todo-tree.general.tags": [
        "BUG",
        "HACK",
        "FIXME",
        "TODO",
        "XXX"
    ]
```
